src/ui/server/controllers/insurance/feedback/feedback-form/index.test.ts

```diff
@@ -0,0 +1,175 @@
+import { TEMPLATE, pageVariables, MAXIMUM, get, post } from '.';
+import { PAGES, FIELDS } from '../../../../content-strings';
+import { TEMPLATES, ROUTES, FIELD_IDS } from '../../../../constants';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import { mockReq, mockRes } from '../../../../test-mocks';
+import { Request, Response } from '../../../../../types';
+import api from '../../../../api';
+import generateValidationErrors from './validation';
+
+const { SATISFACTION, IMPROVEMENT, OTHER_COMMENTS, VERY_SATISFIED, SATISFIED, NEITHER, DISSATISFIED, VERY_DISSATISIFED } = FIELD_IDS.FEEDBACK;
+
+const { FEEDBACK_PAGE } = PAGES;
+const { FEEDBACK: FEEDBACK_TEMPLATE } = TEMPLATES.INSURANCE;
+
+const { FEEDBACK_SENT } = ROUTES.INSURANCE;
+
+describe('controllers/insurance/feedback/feedback-confirmation', () => {
+  let req: Request;
+  let res: Response;
+
+  const mockFlash = jest.fn();
+
+  beforeEach(() => {
+    req = mockReq();
+    req.flash = mockFlash;
+
+    res = mockRes();
+  });
+
+  describe('TEMPLATE', () => {
+    it('should have the correct template defined', () => {
+      expect(TEMPLATE).toEqual(FEEDBACK_TEMPLATE);
+    });
+  });
+
+  describe('pageVariables', () => {
+    it('should have correct properties', () => {
+      const result = pageVariables();
+
+      const expected = {
+        FIELDS: {
+          SATISFACTION: {
+            ID: SATISFACTION,
+            VALUE_ID: { VERY_SATISFIED, SATISFIED, NEITHER, DISSATISFIED, VERY_DISSATISIFED },
```

Comment:
What's the reason for `VALUE_ID`? Could we name it `OPTIONS` object instead so it's aligned with other radios?